pkg/config/config.go

```diff
@@ -29,6 +31,17 @@ type CodeFlareOperatorConfiguration struct {
 	ControllerManager `json:",inline"`
 
 	KubeRay *KubeRayConfiguration `json:"kuberay,omitempty"`
+
+	AppWrapper *AppWrapperConfiguration `json:"appwrapper,omitempty"`
+}
+
+type AppWrapperConfiguration struct {
+	// Enabled controls whether or not the AppWrapper Controller is enabled
+	Enabled *bool `json:"enabled,omitempty"`
+
+	// AppWrapper contains the AppWrapper controller configuration
+	// +optional
+	Config awconfig.AppWrapperConfig `json:"config,omitempty"`
```

Comment:
Sorry I may not have been clear enough, here I was suggesting ``json:",inline"``, WDYT?

Reply:
got it now.  I missed it the first time. That makes sense.  Making the change now.